### ISSUE TITLE
[$40]Challenge Details: Source file formats section is missing

### ISSUE THREAD

**Issue by SathyaJayabal** [topcoder-platform/community-app#4482]
V5 URL:https://beta-community-app.topcoder.com/challenges/5776a168-9c4a-43ea-9a69-a5f369777b3a
V4 URL:https://www.topcoder.com/challenges/30127937
Challenge Details: Source file formats section is missing
<img width="1440" alt="Screenshot 2020-06-07 at 12 21 24 PM" src="https://user-images.githubusercontent.com/58783823/83962265-b823ee00-a8b9-11ea-90a1-41b260821876.png">
<img width="1440" alt="Screenshot 2020-06-07 at 12 22 03 PM" src="https://user-images.githubusercontent.com/58783823/83962273-cbcf5480-a8b9-11ea-97bd-c69c7317a9fd.png">

**Comment by sushilshinde** [comment]
@topcoder-platform/topcodercompetitors @topcoder-platform/tcxcommunity this is open for pickup

Submit in 8 hours and PR to

https://github.com/topcoder-platform/community-app/tree/integration-v5-challenge-api
https://github.com/topcoder-platform/topcoder-react-lib/tree/integration-v5-challenge-api

run `npm run test` before PR

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30128606 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30128606 has been updated - it has been assigned to rashmi73.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by rashmi73** [comment]
@sushilshinde @rootelement here is PR https://github.com/topcoder-platform/community-app/pull/4487

also make sure metadata array has object present with name "fileTypes" in v5 api, because currently it is not present in v5 response and also once available in v5 it wont be visible unless my PR is merged because "type" is renamed to "name" in metadata.

**Comment by simranb86** [comment]
> @sushilshinde @rootelement here is PR #4487
> 
> also make sure metadata array has object present with name "fileTypes" in v5 api, because currently it is not present in v5 response and also once available in v5 it wont be visible unless my PR is merged because "type" is renamed to "name" in metadata.

@rashmi73 test with http://api.topcoder-dev.com/v5/challenges?legacyId=30048648 

@SathyaJayabal FYI too for testing ☝️

**Comment by SathyaJayabal** [comment]
@rashmi73 @sushilshinde ,we are getting a blank page on loading challenge details page for design contests, with error related to file types. Please see screenshot.

The api does return the "fileTypes" in the meta data array.
http://api.topcoder-dev.com/v5/challenges?legacyId=30048648
http://api.topcoder.com/v5/challenges?legacyId=30127937
<img width="1440" alt="Screenshot 2020-06-08 at 10 15 10 AM" src="https://user-images.githubusercontent.com/58783823/83993619-8b331200-a971-11ea-8f48-d3357f5f84a8.png">

**Comment by sushilshinde** [comment]
@rashmi73 please patch

**Comment by rashmi73** [comment]
@sushilshinde patching

**Comment by sushilshinde** [comment]
Reverting code.

**Comment by rashmi73** [comment]
@sushilshinde that failure is due to my code?

**Comment by sushilshinde** [comment]
Yes

**Comment by sushilshinde** [comment]
@rashmi73 are you sending PR? please do that 4 hours

**Comment by rashmi73** [comment]
@sushilshinde it is completed I just forgot to mention it here PR https://github.com/topcoder-platform/community-app/pull/4489

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30128606<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by sushilshinde** [comment]
@rashmi73 please PR this again, I had to revert for other reason.

nothing to do with you code.

**Comment by rashmi73** [comment]
@sushilshinde sure

**Comment by rashmi73** [comment]
@sushilshinde here is PR again https://github.com/topcoder-platform/community-app/pull/4508

**Comment by SathyaJayabal** [comment]
> @SathyaJayabal @lakshmiathreya, can you close this one yet?

@Oanh-and-only-Oanh , this was also reverted with the rest on tuesday. New PR is merged now- We will test and close it.

**Comment by SathyaJayabal** [comment]
verified on test env

<img width="1440" alt="Screenshot 2020-06-11 at 8 59 57 AM" src="https://user-images.githubusercontent.com/58783823/84341968-0ab72000-abc2-11ea-973b-ada4e9bf1919.png">
